src/ethereum/vm/gas.py

```diff
@@ -19,6 +19,10 @@
 GAS_STORAGE_SET = U256(20000)
 GAS_STORAGE_UPDATE = U256(5000)
 GAS_STORAGE_CLEAR_REFUND = U256(15000)
+GAS_LOW = U256(5)
+GAS_MID = U256(8)
+GAS_EXP = U256(10)
+GAS_EXP_BYTE = U256(10)
```

Comment:
Is there something that could possibly be more descriptive for this? Since it's the same cost for `GAS_EXPONENTIATION`, could we just use that one?

Reply:
I prefer keeping them separate as it is very likely that these values may diverge in the future. I would name it `GAS_EXPONENTIATION_BYTE`. It's hard to come up with a descriptive name for this variable.

Reply:
If they diverge in the future, wouldn't it make more sense to then introduce a new variable?

Reply:
Ok, I'll use the same variable for both the scenarios.